Makefile

```diff
@@ -54,17 +62,11 @@ clear-caches:
 
 .PHONY: setup
 setup: .env
-	docker-compose run web /app/bin/setup-services.sh
+	docker-compose run --rm web /app/bin/setup-services.sh
```

Comment:
This was leaking docker containers. This should reduce the number of containers created and orphaned.